infrastructure/resource_group_level/modules/function-acbs.bicep

```diff
@@ -101,15 +99,6 @@ resource functionAcbs 'Microsoft.Web/sites@2022-09-01' = {
       scmMinTlsVersion: '1.0'
       remoteDebuggingVersion: 'VS2019'
       httpLoggingEnabled: true // false in staging
-      // TODO:FN-684 Note that the following appear in dev but not staging or prod. Remove if not needed.
```

Comment:
As mentioned in the thread in Slack, I don't think these are needed. Please confirm as this will update the infrastructure.